src/app/model/model-merging-model.ts

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2021 University of Oxford
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+import { ModelDomainType } from './model-domain-type';
+
+export class ModelMergingModel {
+  sourceId: string;
+  targetId: string;
+  catalogueDomainType: ModelDomainType;
+
+  constructor(sourceId, targetId,catalogueDomainType)
```

Comment:
A `class` this simple could be simplified further. Either it can just become an `interface` - meaning no constructor is required at all, so long as creating an object literal is OK for this use case - or the class can be simplified to this:

```ts
export class ModelMergingModel {
  constructor(public sourceId: string, public targetId: string, public catalogueDomainType: ModelDomainType) { }
}
```